apps/frontend/src/app/PageArtifact/ArtifactEditor.tsx

```diff
@@ -254,12 +254,11 @@ export default function ArtifactEditor({ artifactIdToEdit = "", cancelEdit, allo
               {/* Artifact Set */}
               <ArtifactSetAutocomplete
                 disabled={disableSet}
-                disableClearable
                 size="small"
                 artSetKey={artifact?.setKey ?? ""}
                 setArtSetKey={updateSetKey}
                 sx={{ flexGrow: 1 }}
-                label={t("editor.unknownSetName")}
+                label={artifact?.setKey ? "" : t("editor.unknownSetName")}

```

Comment:
Think the proper way to fix this is to expose "placeholder" prop in GeneralAutocomplete that gets used in renderInput. So label should always be "Artifact Set Name" and placeholder can be "Unknown Artifact Set". But I am also ok with this fix too

Reply:
this component doesnt really show a "placeholder", since the title occupies the space when its empty...